README.md

```diff
@@ -2,139 +2,140 @@
-            doc: |
-               A single block of doc string.
-      ```
-      Such structure of doc would appear in `nxdl` as
-      ```xml
-      ...
-      <field name="energy">
-        <doc>
-            Part1 of the entire doc.
-            part1 of the entire doc.
-
-                This concept is related to term `&lt;term&gt;`_ of the &lt;spec&gt; standard.
-            .. _&lt;term&gt;: &lt;url&gt;
-
-            Rest of the doc
-            rest of the doc
-        </doc>
-      </field>
-      <field name="velocity">
-        <doc>
-            A single block of doc string.
-        </doc>
-      </field>
-      ```
-
-
-
-* **exists** Options are recommended, required, [min, 1, max, `infty`] numbers like here 1 can be replaced by any `uint` (unsigned integer), or `infty` to indicate no restriction on how frequently the entry can occur inside the NXDL schema at the same hierarchy level.
-* **link** Define links between nodes.
-* **units** A statement introducing NeXus-compliant NXDL units arguments, like NX_VOLTAGE
-* **dimensions** Details which dimensional arrays to expect
-* **dim** Shorthand notation for dimensions, e.g., `(n, )` for an 1D array of length `n` or `(n, m)` for an 2D array of size `n x m`.
-* **enumeration** Python list of strings which are considered as recommended entries to choose from.
-* **dim_parameters** `dim` which is a child of `dimension` and the `dim` might have several attributes `ref`,
-`incr` including `index` and `value`. So while writing `yaml` file schema definition please following structure:
 ```yaml
-dimensions:
-   rank: integer value
-   dim: [[ind_1, val_1], [ind_2, val_2], ...]
-   dim_parameters:
-      ref: [ref_value_1, ref_value_2, ...]
-      incr: [incr_value_1, incr_value_2, ...]
+energy:
+  doc:
+    - |
+      Part1 of the entire doc.
+      part1 of the entire doc.
+    - |
+      xref:
+        spec: <spec>
+        term: <term>
+        url: <url>
+    - |
+      Rest of the doc
+      rest of the doc
+  velocity:  # field
+    doc: |
+        A single block of doc string.
 ```
-Keep in mind that length of all the lists must have the **same size**.
-**Important Note**: The attributes `ref`, `incr`, `index` are deprecated.
 
-## Project roadmap
+This structure appears in Nxdl as follows:
+
+```xml
+<field name="energy">
+  <doc>
+    Part1 of the entire doc.
+    part1 of the entire doc.
+    This concept is related to term `<term>`_ of the `<spec>` standard.
+    .. _`<term>`: `<url>`
+    Rest of the doc
+    rest of the doc
+  </doc>
+  <field name="velocity">
+    <doc>
+        A single block of doc string.
+    </doc>
+  </field>
+</field>
+```
+
+### Additional Keywords
+
+- **exists**: Options for `exists` can be recommended, required, or specified as `[min, 1, max, infty]`. Replace `1` with any unsigned integer or use `infty` to indicate no restriction on occurrence within the Nxdl schema at the same hierarchy level.
+
+- **link**: Defines links between nodes.
```

Comment:
Add link to description of linkType in NeXus: https://manual.nexusformat.org/nxdl_desc.html#linktype